### PR TITLE
chore: upgrade nextjs 13 and tailwind

### PR DESCRIPTION
**Pull request by Aztriltus**
Changes
- Upgrade from Nextjs 12 to 13 so we don't have to manually update it for future projects
- Upgrade Tailwindcss while we're at it

### CODE REVIEW

**Review by ayungavis — COMMENTED** [review]
I think better to add a bot to automatically create a PR when the version is outdated. What do you think? 🤔
```yml
version: 2
updates:
  - package-ecosystem: npm
    directory: '/'
    schedule:
      interval: daily
    open-pull-requests-limit: 2
```

Reference:
https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

**Comment by Aztriltus** [comment]
> I think better to add a bot to automatically create a PR when the version is outdated. What do you think? 🤔
> 
> ```yaml
> version: 2
> updates:
>   - package-ecosystem: npm
>     directory: '/'
>     schedule:
>       interval: daily
>     open-pull-requests-limit: 2
> ```
> 
> Reference: https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

That's a good suggestion; I didn't know about that. But I think sometimes dependabot gets quite irritating when we don't want to risk the upgrades, especially when we have production apps or we need to use libs that might use older dependencies. If that's the case, would you still recommend adding it? @ayungavis

**Comment by Aztriltus** [comment]
Had to remove building with Node v14 git workflow since it's not compatible with the latest nextjs with sass. I think it should be fine to use only Node v16

File: package.json [hunk 1]
```diff
@@ -16,25 +16,25 @@
   "dependencies": {
     "@typescript-eslint/eslint-plugin": "^5.26.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
```
Comment (sushanyadav) [inline package.json]: Since we are here, should we move these to dev dependencies?

**Review by sushanyadav — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Aztriltus) [reply package.json]: Yes sounds good

**Comment by ayungavis** [comment]
> That's a good suggestion; I didn't know about that. But I think sometimes dependabot gets quite irritating when we don't want to risk the upgrades, especially when we have production apps or we need to use libs that might use older dependencies. If that's the case, would you still recommend adding it? @ayungavis

Ahh yeah, I think we upgrade it manually like this for now 😅

**Review by ayungavis — APPROVED** [review]

**Pull request merged**